src/state/actions/scigateway.actions.test.tsx

```diff
@@ -52,6 +52,8 @@ function mockAxiosGetResponse(message: string): void {
 
 describe('scigateway actions', () => {
   beforeEach(() => {
+    jest.useFakeTimers();
```

Comment:
Worth adding this?

```
afterEach(() => {
    jest.runOnlyPendingTimers();
    jest.useRealTimers();
});
```

Good practice according to https://testing-library.com/docs/using-fake-timers/

Reply:
Good spot - I'll add this in and add it to other test files that also use fake timers 👍